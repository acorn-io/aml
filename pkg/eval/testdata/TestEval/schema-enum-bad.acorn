schema {
    a: enum("a", "b", "c")
}

a: "f"
