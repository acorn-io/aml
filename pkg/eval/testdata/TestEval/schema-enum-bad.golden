`schema violation .a: option 1: [unmatched constraint f == c],
option 2: [unmatched constraint f == b],
option 3: [unmatched constraint f == a] (schema-enum-bad.acorn:1:8)`
